assets/app.js

```diff
@@ -553,15 +555,17 @@ new Vue({
             }
         },
         translatorBaseProducts: function(){
-            if (BaseProductOptions && BaseProductOptions.content) {
-                var response = []
-
-                BaseProductOptions.content.forEach(function (option, index) {
-                    if(option.name !== '-' && option.is_active==1 && option.translation_requested == 1){
-                        response.push(option);
-                    }
-                })
-                return response
+            if(this.baseProductStatus){
+                this.baseProductStatus = false
+                if (BaseProductOptions && BaseProductOptions.content) {
```

Comment:
What about get rid off the new condition and combining it with the existing one?
``` 
if (this.baseProductStatus && BaseProductOptions && BaseProductOptions.content) {
  this.baseProductStatus = false
  ....
```

Reply:
what do we get from `translatorBaseProducts` once  this.baseProductStatus == false?

Reply:
the problem is `translatorBaseProducts` is working only first time that we call in template.
when we update status in translation --> `translatorBaseProducts` will return data as first time (No update data)
so  after  ` this.baseProductStatus == false`   it will be assigned true 
` _this.app.baseProductStatus = true` in callback (product_descriptor.config.js) for making `translatorBaseProducts`  is return latest value.

Reply:
### problem: ###
the problem happen when I changed   _translation_requested_  from` 0` to `1`  of `Anklets`(example) on ___translation view___  ( ___admin page___) 
### Expected:  ###
`Anklets` will be show in translator view
### Actually: ###
`Anklets` doesn't show in translator view
### Cause:   ###
`translatorBaseProducts` will be filtered data only first time which we call it, other will return data as first time  that means if we switch requested_status = 1 from  admin page then it will not add any data in `translatorBaseProducts` but we can switch translation requested.

So `translatorBaseProducts` cannot ADD data but can SWITCH  `translation_requested` by `switchTranslationStatus` or `switchTranslatorStatus`

### Resolve  ###
######  `Goal` ###### 
Make __translatorBaseProducts__   filter data every time which be called.
###### `plan` ###### 
if some variable of data properties which used in  __translatorBaseProducts__  are changed value then __translatorBaseProducts__  will filter data .

######  `What I did` ###### 
I set __this.baseProductStatus = true__ for working in first time and then set __false__ when it's done. 
next step I set __this.baseProductStatus = true__  in callback to update and filter data in  __translatorBaseProducts__.

`what do we get from translatorBaseProducts once this.baseProductStatus == false?`
We will get data which filterd in latest callback.

Reply:
So `baseProductStatus` holds the information if we need to reload Base Products?
- call the variable `reloadBaseProducts` (**false** by default, set to **true** after the first time reading it)
- set back to **false** on callback